bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -26,6 +26,9 @@ class CategoricalAxis extends Axis.Model
 
     return range_bounds
 
+  display_defaults: () ->
+    return super()
+
```

Comment:
I don't follow right now, but it's just too late in the day. I will get back to this tomorrow.